docs/configurations/kafkacluster/examples/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Kafka cluster
+shorttitle: KafkaCluster examples
+weight: 7000
+---
+
+The following KafkaCluster custom resource examples show you some basic use cases.
+You can use these examples as a base to your own Kafka cluster.
+
+> Note: These examples can be used if you are using Koperator directly. If you are using Calisti Streaming Data Manager, you must adjust the examples to match the requirements of Calisti. For details, see the [Calisti Streaming Data Manager documentation](https://docs.calisti.app/docs/sdm/create-kafka-cluster/).
```

Comment:
```suggestion
> Note: These examples can be used if you are using Koperator directly. If you are using [Calisti Streaming Data Manager](https://docs.calisti.app/sdm/), you must adjust the examples to match the requirements of Calisti. For details, see the [Calisti Streaming Data Manager documentation](https://docs.calisti.app/docs/sdm/create-kafka-cluster/).
```

Reply:
I would also rather state this on the Calisti docs instead of here, but I can be convinced otherwise

Reply:
It can be good when a Calisti user got lost in the https://docs.calisti.app/sdm/  doc and he wants to create his kafkacluster cr based on the koperator/configuration/kafkacluster.

Reply:
I think in my other PR it is clear for the user what has to be used to create his kafkacluster for calisti.
So I think this note can be removed.
What others think?

Reply:
I have removed this.